onnxruntime/core/providers/cpu/rnn/uni_directional_lstm.h

```diff
@@ -109,6 +109,17 @@ class UniDirectionalLstm {
   ActivationInfo<deepcpu::LstmMergeGatesFuncPtr> activation_h_;
 
   concurrency::ThreadPool* thread_pool_;
+
+  // Quantized operation related allocation members
+  template <typename WeightT>
+  void AllocateQuantizeBuffers(int max_sequence_length);
+
+  // Buffer shared for quantized input whole, and quantized a each sequence step
+  IAllocatorUniquePtr<uint8_t> quantized_input_or_a_ptr_;
+  gsl::span<uint8_t> quantized_input_or_a_;
+
+  IAllocatorUniquePtr<int32_t> quantize_agg_C_ptr_;
```

Comment:
nit: can you ask Yufeng for a better name for this? I'm not sure why it's "agg". Would be nice if it were quantized_ to match the above pointer too

Reply:
how about just quantized_C_buffer_ptr_ and quantized_C_buffer_ for the gsl::span. 

It is used to holder the int32 result of MatMul

Reply:
No problem